app/controllers/image_sets_controller.rb

```diff
@@ -34,6 +32,27 @@ def destroy
 
   private
 
+  def search_results
+    if search_params.has_key?(:lions)
+      ImageSet.joins(:lion).where(search_params)
+    else
+      ImageSet.where(search_params)
+    end
+  end
+
+  def search_params
+    _params = params.permit(:age, :gender, :name, :organization_id)
+    if owner_organization_id = _params[:organization_id]
+      _params = _params.except(:organization_id).merge({owner_organization_id: owner_organization_id})
```

Comment:
what does this do? You are changing the name of the key from `:organization_id` to `:owner_organization_id`? It's a little hard to understand that from reading the code. Something like this would probably be easier to mentally parse:
```
if _params[:organization_id]
  _params[:owner_organzation_id] = params.delete(:organization_id)
end
```